package.json

```diff
@@ -187,6 +187,7 @@
     "@shopify/app-bridge": "^0.7.3",
     "@shopify/images": "^1.1.0",
     "@shopify/javascript-utilities": "^2.2.1",
+    "@shopify/polaris-icons": "^1.0.0-beta.2",
```

Comment:
Maybe it's just me being conservative, but we may wanna pin this to a strict version (no `^`) until we've reached 1.0.0.